docs/node/production-guide.mdx

```diff
@@ -0,0 +1,392 @@
+
+```sh title="node1"
+[node1]:~$ kwil-admin setup init --genesis prod-chain/genesis.json \
+  --chain.moniker "node1" \
+  --chain.p2p.external-address "98.76.54.32:26656" --chain.p2p.persistent-peers \
+    "2a8b9eb0d94be766ad34cb2591f2924477fc397c@23.45.67.89:26656" \
+  --root-dir /data/kwil-node1
+Initialized node in /data/kwil-node1
+```
+
+#### Initial Startup and Block Production
+
+Now that both of the validators have been prepared with the same `genesis.json` file and their persistent peer configurations pointing to each other, it is time to start the nodes.
+
+:::tip
+Depending on how the nodes are deployed, `kwild` would be run as a service, container, or in a terminal multiplexer.
+:::
+
+Start the nodes on each of the remote machines, specifying the path to the root directory if it is not at the default location (`~/.kwild`):
+
+```sh title="node0"
+[node0]:~$ kwild -r /data/kwil-node0
+```
+
+```sh title="node1"
+[node1]:~$ kwild -r /data/kwil-node1
+```
+
+Once the nodes connect, the first block will be produced shortly after:
+
+```plain
+2024-09-19T12:01:23.443-05:00   info    kwild.cometbft  finalizing commit of block      {"module": "consensus", "height": 1, "hash": "010806C959A3CB3F186C1216745E9C9C22F38F98A8D6C8673848BFF79FBD6C81", "root": "B4D2AE49E40746CB1C5C1F1B89D3B30F4927A566E0E0D7F7EDBC6C5A1F540B18", "num_txs": 0}
+2024-09-19T12:01:23.465-05:00   info    kwild.cometbft  finalized block {"module": "state", "height": 1, "num_txs_res": 0, "num_val_updates": 0, "block_app_hash": "5DF6E0E2761359D30A8275058E299FCC0381534545F55CF43E41983F5D4C9456"}
+```
+
+Recall that *both* of the validators need to be started and able to communicate before blocks can be created, as discussed in the [Validators section](#validators). If the nodes are unable to connect, look for peer-related errors in the logs, and check your persistent peers settings and firewall rules.
+
+### Adding a New Node
+
+Now that our network of two validators is running and producing blocks, we will created and synchronize a new node to the network, and once it is synchronized, promote the node to a validator.
+
+#### Starting as a Sentry Node
+
+Now we will add a third note to the network as a sentry node. Since this node is not a genesis validator, its public key is not included in the genesis file, which is now immutable. The `kwil-admin setup init` command can be used to simultaneously initialize a new node's root directory and generate a new private key.
+
+:::important
+When adding a node to the network, start it as a non-validator / sentry node.
+Always completely synchronize the node to the network before attempting to promote it to a validator.
+:::
+
+Like before we will specify the location of the network's `genesis.json` file, the host's external IP address, and one or more persistent peers. However, we will *not* specify a private key path so that a new key is generated:
```

Comment:
```suggestion
Like before, we will specify the location of the network's `genesis.json` file, the host's external IP address, and one or more persistent peers. However, we will *not* specify a private key path so that a new key is generated:
```